recipes/libwebm/all/conandata.yml

```diff
@@ -0,0 +1,14 @@
+sources:
+  '1.0.0.29':
+    url: 'https://chromium.googlesource.com/webm/libwebm/+archive/2f9fc054ab9547ca06071ec68dab9d54960abb2e.tar.gz'
```

Comment:
could you please use the tag instead? it's available in the repo and is easier to identify.

Plus, is it possible to package 1.0.0.30 instead of 29?

Reply:
Yes, I was waiting for them to create the new tag and have had a long weekend.

Reply:
Thank you!